packages/grafana-e2e/src/flows/importDashboard.ts

```diff
@@ -50,7 +50,7 @@ export const importDashboard = (dashboardToImport: Dashboard, queryTimeout?: num
     e2e.components.Panels.Panel.title(panel.title).should('be.visible').click();
     e2e.components.Panels.Panel.headerItems('Inspect').should('be.visible').click();
     e2e.components.Tab.title('JSON').should('be.visible').click();
-    e2e.components.PanelInspector.Json.content().should('be.visible').contains('Panel JSON').click();
+    e2e.components.PanelInspector.Json.content().should('be.visible').contains('Panel JSON').click({ force: true });
```

Comment:
do we know why this is necessary? doesn't look like anything in the changes would have caused this, but i guess it must have 😕

Reply:
because the input is ontop of the value and cypress will complain that the value is not visible as it's covered by the input